openwisp_radius/apps.py

```diff
@@ -104,13 +105,61 @@ def radiusorgsettings_post_save(self, instance, **kwargs):
     def radiusorgsettings_post_delete(self, instance, **kwargs):
         instance.delete_cache()
 
-    def add_default_menu_items(self):
-        menu_setting = 'OPENWISP_DEFAULT_ADMIN_MENU_ITEMS'
-        items = [
-            {'model': f'{self.label}.RadiusAccounting', 'label': _('Radius sessions')}
-        ]
-        if not hasattr(settings, menu_setting):
-            setattr(settings, menu_setting, items)
-        else:
-            current_menu = getattr(settings, menu_setting)
-            current_menu += items
+    def regiser_menu_groups(self):
+        register_menu_group(
+            position=30,
+            config={
+                'label': _('FREERADIUS'),
+                'items': {
+                    1: {
+                        'label': _('Accountings'),
+                        'model': get_model_name(self.label, 'RadiusAccounting'),
+                        'name': 'changelist',
+                        'icon': 'ow-radius-accounting',
+                    },
+                    2: {
+                        'label': _('Batch user Creation'),
+                        'model': get_model_name(self.label, 'RadiusBatch'),
+                        'name': 'changelist',
+                        'icon': 'ow-batch-creation',
+                    },
+                    3: {
+                        'label': _('Checks'),
+                        'model': get_model_name(self.label, 'RadiusCheck'),
+                        'name': 'changelist',
+                        'icon': 'ow-radius-checks',
+                    },
+                    4: {
+                        'label': _('Groups'),
+                        'model': get_model_name(self.label, 'RadiusGroup'),
+                        'name': 'changelist',
+                        'icon': 'ow-radius-group',
+                    },
+                    5: {
+                        'label': _('NAS'),
+                        'model': get_model_name(self.label, 'Nas'),
+                        'name': 'changelist',
+                        'icon': 'ow-radius-nas',
+                    },
+                    6: {
+                        'label': _('Post Auth Log'),
+                        'model': get_model_name(self.label, 'RadiusPostAuth'),
+                        'name': 'changelist',
+                        'icon': 'ow-radius-post-log',
+                    },
+                    7: {
+                        'label': _('Radius Token'),
+                        'model': get_model_name(self.label, 'RadiusToken'),
+                        'name': 'changelist',
+                        'icon': 'ow-radius-token',
+                    },
```

Comment:
we don't need to show this I believe, it's not useful to end users but only to us developers. Maybe we can show it only if DEBUG is True, what do you think @atb00ker?

Reply:
Yes, that sounds good to me! 😄